antora-playbook.yml

```diff
@@ -10,7 +10,15 @@ site:
   title: Couchbase Docs
   url: https://docs.couchbase.com
   start_page: home::index.adoc
-  robots: allow
+  robots: |
+    User-agent: *
+    Allow: /
+    Disallow: /sdk-api/
+    Allow: /sdk-api/*-client/
+    Allow: /sdk-api/*-core-api/
+    Allow: /sdk-api/couchbase-transactions-java/
```

Comment:
No need to crawl any of these txns libraries now, I think - but I'll check to be sure.

Reply:
(Current status - Java transactions separate API link no longer used; others to follow later)